offchain/hello-world-browser/src/Main.purs

```diff
@@ -71,31 +71,45 @@ main :: Effect Unit
 main =
   HA.runHalogenAff do
     body <- HA.awaitBody
-    contractConfig <- useKeyWallet >>= case _ of
-      true -> do
-        configParams <- getConfigParams
-        walletSpec <- loadWalletSpec
-        networkId <- loadNetworkId
-        -- the mainnetConfig is defined as `testnetConfig { networkId = MainnetId }` in CTL
-        pure $ configParams
-          { walletSpec = Just walletSpec
-          , networkId = networkId
-          }
-      false -> pure $ testnetNamiConfig { logLevel = Warn }
+
+    -- the mainnetConfig is defined as `testnetConfig { networkId = MainnetId }` in CTL
+    -- see https://github.com/Plutonomicon/cardano-transaction-lib/blob/886ac0a08989e5c4928f907f3f86448e8836c799/src/Contract/Config.purs#L86
+    configParams <- getConfigParams
+
+    -- We load the walletspec and network id
+    -- from cookies for testing.
+    -- if we don't use the keyWallet we
+    -- assume we are in production
+    walletSpec <- useKeyWallet >>= case _ of
+      true -> loadWalletSpecFromCookie
+      false -> pure ConnectToNami
+
+    networkId <- useKeyWallet >>= case _ of
+      true -> loadNetworkIdFromCookie
+      false -> pure MainnetId
```

Comment:
We could use TestnetId initially, when the page loads, it will fetch the real network ID with FFI to Nami

```suggestion
      false -> pure TestnetId
```